### PR TITLE
MBS-12617, MBS-12618, MBS-12627 - adding new entity icons

### PR DESCRIPTION
**Pull request by Aerozol**
Adds png and svg icon files for:
Collections, tags, folksonomy tags

### CODE REVIEW

**Review by reosarevok — CHANGES_REQUESTED** [review]
Not sure what happened here with the extra commits, maybe you can drop them? :) 

This adds the icons but does not use them. You should add them to `root/static/styles/entity-icons.less`. I can always do that in a second commit though if that seems annoying.

One thing we do need is the inverted icons for the links :) Any chance you could add those? (see the same folder). IIRC @mwiencek created the previous ones, if it's not trivial maybe he has a way saved to do so.

**Review by yvanzo — APPROVED** [review]
Just had a working session with @Aerozol who made the inverted icons while I added the code changes and tested the whole thing a bit:

* Page header:
  * Collection: straightforward
  * Genre: straightforward
  * Tag: added a subheader (empty so far) to match the icon height
* Link display:
  * Collection: straightforward but neither actually used nor needed so far
  * Genre: straightforward and used for genre relationships
  * Tag: added the Less code only but the CSS class is neither currently added nor needed so far

**Review by reosarevok — APPROVED** [review]
For tag, we could possibly show the icon in pages like https://musicbrainz.org/genre/bf44ad8f-47be-4bcc-b577-ba2268160cd1 (for "Primary tag").

But this and the other comment are minor and this can be merged as-is if preferred, assuming you have tested the display.

File: root/tag/TagLayout.js [hunk 1]
```diff
@@ -48,9 +49,12 @@ const TagLayout = ({
...
   >
     <div id="content">
-      <h1>
-        {exp.l('Tag “{tag}”', {tag: <TagLink tag={tag.name} />})}
-      </h1>
+      <div className="tagheader">
+        <h1>
+          {exp.l('Tag “{tag}”', {tag: <TagLink tag={tag.name} />})}
+        </h1>
+        <SubHeader subHeading="" />
```
[the remark this answers is not in the document]
Comment (yvanzo) [reply root/tag/TagLayout.js]: Yes, now changed in https://github.com/metabrainz/musicbrainz-server/pull/2997/commits/a1ad396193941ca3888a3f5ea67071d436eee018#diff-9ec4db304fc650799a4cb07ce150d4b51783bf99fd47dc1025a084b761c11b46R56.

**Comment by yvanzo** [comment]
> For tag, we could possibly show the icon in pages like https://musicbrainz.org/genre/bf44ad8f-47be-4bcc-b577-ba2268160cd1 (for "Primary tag").

Now done in https://github.com/metabrainz/musicbrainz-server/pull/2997/commits/f9002872a2f310be726714a0418bc5d4f08cac83.

Thanks for the review!

**Review by reosarevok — APPROVED** [review]
I'm sure we'll find a few more places where to display icons later, but let's start with this, thanks.

**Comment by yvanzo** [comment]
I thought about the sidebar but the section “Genres” actually contains tag links. It seems to be a separate issue.

**Comment by yvanzo** [comment]
@brainzbot, please retest this.

**Pull request merged**